src/Visualization/Python/Render1D.py

```diff
@@ -0,0 +1,274 @@
+        assert len(h5_file_names) > 0,\
+            "Found no files with prefix {} in directory {}"\
+            .format(files, os.getcwd())
+        h5_files = [h5py.File(file_name, 'r') for file_name in h5_file_names]
+
+    else:
+        for file_name in files:
+            try:
+                h5_files.append(h5py.File(file_name, 'r'))
+            except IOError as err:
+                sys.exit("Could not find file '{}' in directory {}: {}".format(
+                    file_name, os.getcwd(), err))
+    return h5_files
+
+
+def print_var_names(files):
+    '''
+    Print all available variables to screen
+    :param args:
+    :return: None
+    '''
+
+    h5files = get_h5_files(files)
+    volfile = h5files[0]["element_data.vol"]
+    obs_id_0 = next(iter(volfile))
+    variables = list(volfile[obs_id_0].keys())
+    variables.remove("connectivity")
+    variables.remove("InertialCoordinates_x")
+    variables.remove("total_extents")
+    variables.remove("grid_names")
+    print("Variables in H5 file:\n{}".format(list(map(str, variables))))
+    for h5_file in h5files:
+        h5_file.close()
+
+
+def get_data(files, var_name):
+    '''
+    Get the data to be plotted
+    :return: the array of data, coords, and time
+    '''
+
+    time = []
+    coords = []
+    data = []
+    h5files = get_h5_files(files)
+    volfiles = [h5file['element_data.vol'] for h5file in h5files]
+    # Get a list of times from the first vol file
+    ids_times = [(obs_id, volfiles[0][obs_id].attrs['observation_value'])
+                 for obs_id in volfiles[0].keys()]
+    ids_times.sort(key=lambda pair: pair[1])
+    for obs_id, local_time in ids_times:
+        local_coords = []
+        local_data = []
+        ordering = None
+        for volfile in volfiles:
+            try:
+                local_data = (local_data + list(volfile[obs_id][var_name]))
+            except KeyError:
+                print("The variable name {} is not a valid variable. "
+                      "Use '--list-vars' to see the list of variable names in "
+                      "the file(s) \n{}".format(var_name,
+                                                list(map(str, h5files))))
+                sys.exit(1)
+            local_coords = (local_coords +
+                            list(volfile[obs_id]['InertialCoordinates_x']))
+        ordering = np.argsort(local_coords)
+        coords.append(np.array(local_coords)[ordering])
+        data.append(np.array(local_data)[ordering])
+        time.append(local_time)
+
+    for h5file in h5files:
+        h5file.close()
+    return time, coords, data
+
+
+def render_single_time(var_name, time_slice, output_prefix, coords, data):
+    '''
+    Renders a single time snap shot
+    :param args: the command line arguments
+    :return: None
+    '''
+    import matplotlib.pyplot as plt
+
+    plt.xlabel("x")
+    plt.ylabel(var_name)
+    plt.title("t = {:.5f}".format(int(time_slice)))
```

Comment:
Don't convert to `int`. This line currently always displays 5 zeros after the decimal point. Just remove the `int()` so the 5 numbers after the decimal point are actually meaningful, or change the `.5f` to the precision you want.